core/src/test/java/com/netflix/conductor/validations/WorkflowDefConstraintTest.java

```diff
@@ -49,10 +48,7 @@ public class WorkflowDefConstraintTest {
 
     @BeforeClass
     public static void init() {
-        validatorFactory =
-                Validation.byProvider(ApacheValidationProvider.class)
-                        .configure()
-                        .buildValidatorFactory();
+        validatorFactory = Validation.buildDefaultValidatorFactory();
```

Comment:
Need to delete the ApacheValidator because bval-jsr does not allow the new jakarta stuff - and version 3x is not yet public.